fuzz/fuzz_targets/roundtrip.rs

```diff
@@ -66,4 +68,19 @@ fn decode_png(data: &[u8]) -> (png::OutputInfo, Vec<u8>) {
     let info = reader.next_frame(&mut img_data).unwrap();
 
     (info, img_data)
+}
+
+// copied from the `png` codebase because it's pub(crate)
+fn raw_row_length_from_width(depth: BitDepth, color: ColorType, width: u32) -> usize {
+    let samples = width as usize * color.samples();
+    1 + match depth {
```

Comment:
The `1 +` is to account for the filter byte that PNG rows contain before the start of actual image data. It cancels out the `- 1` above.

Reply:
Added comment explaining it. Thanks!